### DISCUSSION TITLE
Programming/Tech Livestreams

### DISCUSSION THREAD

**Discussion by CodeSpent** [dailydotdev/daily#730] [Feature Requests]
I'd like to see programming live-stream discovery also curated on the feed. Tagging based on stream titles could be a semi-reliable way and an entry to how I curated livestreams on an old project, DevStreams.

![image](https://user-images.githubusercontent.com/7674344/193664414-af46c71e-a467-4d3e-8c79-0aeba532f560.png)

As I've had to shut down DevStreams and I really like Daily.dev, I'd be more than willing to contribute on this feature with what I've learned on that venture. Solving my problem again and enhancing the experience on daily.dev as well. :)

**Comment by idoshamun** [comment]
Hi @CodeSpent,

Thanks a lot for the suggestion. We definitely want to get into more content types and surface live streams, videos, podcasts and whatnot. We're still thinking about the right time to do it :)
